<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>9</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#timeLabel {
    font-size: 22px;
    font-weight: bold;
    color: white;
}</string>
   </property>
   <property name="text">
    <string>2:10 AM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#dateLabel {
    font-size: 14px;
    font-weight: medium;
    color: #CCCCCC;
}</string>
   </property>
   <property name="text">
    <string>30 Mar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="insulinBar">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar#insulinBar {
    border: 1px solid #555;
    border-radius: 4px;
    background-color: #111;
    height: 12px;
    text-align: center;
    padding: 0px;
}

QProgressBar#insulinBar::chunk {
    background-color: #00BFFF; 
    width: 4px;
    margin: 1px;
}
</string>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="insulinIcon">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#insulinIcon {
    background-color: #00BFFF;
    color: black;
    font-weight: bold;
    font-size: 12px;
    padding: 2px 6px;
    border-radius: 4px;
}
</string>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="insulinUnitsLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#insulinUnitsLabel {
    color: #88CCDD;
    font-size: 14px;
    font-weight: bold;
    letter-spacing: 0.5px;
}
</string>
   </property>
   <property name="text">
    <string>300 u</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="batteryBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar#batteryBar {
    border: 1px solid #555;
    border-radius: 4px;
    background-color: #222;
    height: 12px;
    text-align: center;
}

QProgressBar#batteryBar::chunk {
    background-color: #00FF00;  
    width: 4px;
    margin: 1px;
}
</string>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="batteryLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#batteryLabel {
    color: #00FF00;
    font-size: 12px;
    font-weight: bold;
    letter-spacing: 1px;
}
</string>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QChartView" name="chartView" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QComboBox" name="comboGraphRange">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1 hour</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 hours</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 hours</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOptions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#optionsButton {
    background-color: #444444;
    color: #FFD700;
    font-weight: bold;
    font-size: 14px;
    padding: 10px;
    border: 1px solid #888;
    border-radius: 5px;
    qproperty-icon: url(:/icons/gear.png);
    qproperty-iconSize: 24px 24px;
}</string>
   </property>
   <property name="text">
    <string>OPTIONS</string>
   </property>
  </widget>
  <widget class="QLabel" name="insulinLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#insulinLabel {
    color: #CCCCCC;
    font-weight: bold;
    font-size: 13px;
    letter-spacing: 1px;
}</string>
   </property>
   <property name="text">
    <string>INSULIN ON BOARD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="iobLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#insulinUnitsLabel_2 {
    color: white;
    font-weight: bold;
    font-size: 14px;
}</string>
   </property>
   <property name="text">
    <string>0 u</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBolus">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#bolusButton {
    background-color: #444444;
    color: #00BFFF;
    font-weight: bold;
    font-size: 14px;
    padding: 10px;
    border: 1px solid #888;
    border-radius: 5px;
    qproperty-icon: url(:/icons/droplet.png);
    qproperty-iconSize: 24px 24px;
}</string>
   </property>
   <property name="text">
    <string>BOLUS</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBolusStatus">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#labelBolusStatus {
    color: #00BFFF;
    font-size: 13px;
    font-weight: bold;
  }</string>
   </property>
   <property name="text">
    <string>No bolus active</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="historyButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#optionsButton {
    background-color: #444444;
    color: #FFD700;
    font-weight: bold;
    font-size: 14px;
    padding: 10px;
    border: 1px solid #888;
    border-radius: 5px;
    qproperty-icon: url(:/icons/gear.png);
    qproperty-iconSize: 24px 24px;
}</string>
   </property>
   <property name="text">
    <string>HISTORY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#logoButton {
    background-color: #005F87;
    color: black;
    font-weight: bold;
    font-size: 16px;
    padding: 6px 12px;
    border-radius: 6px;
    border: 1px solid #007ACC;
}

QPushButton#logoButton:hover {
    background-color: #1E90FF;
}

QPushButton#logoButton:pressed {
    background-color: #007ACC;
    color: white;
}
</string>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QLabel" name="glucoseLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: white;
    font-size: 28px;
    font-weight: bold;
    font-family: &quot;Segoe UI&quot;, sans-serif;
}</string>
   </property>
   <property name="text">
    <string>6.3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #CCCCCC;
    font-size: 14px;
    font-weight: medium;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    margin-top: -10px;
}
</string>
   </property>
   <property name="text">
    <string>mmol/L</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBolusTimeRemaining">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#labelBolusTimeRemaining {
    color: #CCCCCC;
    font-size: 13px;
    font-weight: bold;
  }</string>
   </property>
   <property name="text">
    <string>Time Left: --:--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>line</zorder>
  <zorder>timeLabel</zorder>
  <zorder>dateLabel</zorder>
  <zorder>insulinBar</zorder>
  <zorder>insulinIcon</zorder>
  <zorder>insulinUnitsLabel</zorder>
  <zorder>batteryBar</zorder>
  <zorder>batteryLabel</zorder>
  <zorder>chartView</zorder>
  <zorder>line_2</zorder>
  <zorder>buttonOptions</zorder>
  <zorder>insulinLabel</zorder>
  <zorder>iobLabel</zorder>
  <zorder>buttonBolus</zorder>
  <zorder>labelBolusStatus</zorder>
  <zorder>historyButton</zorder>
  <zorder>logoButton</zorder>
  <zorder>glucoseLabel</zorder>
  <zorder>label_2</zorder>
  <zorder>labelBolusTimeRemaining</zorder>
  <zorder>line_3</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
